src/Microsoft.Data.SqlClient/tests/tools/Microsoft.DotNet.XUnitExtensions/Microsoft.DotNet.XUnitExtensions.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp2.1;net46</TargetFrameworks>
```

Comment:
TargetFramework should not be hard-coded here.

Use variables $(TargetNetFxVersion) and $(TargetNetCoreVersion) that will get values from here:
https://github.com/dotnet/SqlClient/blob/master/src/Microsoft.Data.SqlClient/tests/Directory.Build.props#L9-L10